tests/taskgraphs/test_server_side.py

```diff
@@ -19,6 +19,7 @@
 
 
 class ConnectToExistingTest(unittest.TestCase):
+    @pytest.mark.xfail(reason="Time to complete is not predictable.")
```

Comment:
This test is once again not completing in time. I don't think that increasing the wait is a good strategy, it delays all our builds.